### PR TITLE
[MRG] Fix review comments and enforce 3-D inputs

### PR DESCRIPTION
**Pull request by PraveenVenugopal**
The pull request is ready to be merged. 

Fix review comments and allow only 3-D time series inputs from the user for all RNN blocks

### CODE REVIEW

File: autokeras/layer_utils.py [hunk 1]
```diff
@@ -26,6 +26,10 @@ def format_inputs(inputs, name=None, num=None):
     return inputs
 
 
+def get_block(choice):
```
Comment (jhfjhfj1) [inline autokeras/layer_utils.py]: rename to get_rnn_block

File: autokeras/hypermodel/hyper_block.py [hunk 2]
```diff
@@ -69,34 +70,22 @@ class RNNBlock(HyperBlock):
 
     def build(self, hp, inputs=None):
         input_node = layer_utils.format_inputs(inputs, self.name, num=1)[0]
+        feature_size = input_node.shape[2]
```
Comment (jhfjhfj1) [inline autokeras/hypermodel/hyper_block.py]: add a check first, if len(input_node.shape) < 3: raise ValueError('The input to RNNBlock must have at least 3 dimensions').

File: tests/test_hyper_block.py [hunk 3]
```diff
@@ -3,8 +3,8 @@
 from autokeras.auto.auto_model import *
 from autokeras import const
 
-from autokeras.hypermodel.hyper_block import XceptionBlock, RNNBlock
-from autokeras.hypermodel.hyper_head import ClassificationHead, RegressionHead
+from autokeras.hypermodel.hyper_block import *
+from autokeras.hypermodel.hyper_head import *
```
Comment (jhfjhfj1) [inline tests/test_hyper_block.py]: This is test_hyper_block so we only do hyper_block import *, but not import * for other modules.

File: autokeras/hypermodel/hyper_block.py (hunk 2 above)
Comment (PraveenVenugopal) [reply autokeras/hypermodel/hyper_block.py]: Should we check for < 3 or strictly == 3. To enforce 3-D shape i believe we need the second

File: autokeras/layer_utils.py (hunk 1 above)
Comment (PraveenVenugopal) [reply autokeras/layer_utils.py]: Sure.

File: tests/test_hyper_block.py (hunk 3 above)
Comment (PraveenVenugopal) [reply tests/test_hyper_block.py]: Okay. got it

**Pull request merged**